lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -224,11 +224,16 @@ class CategoryRenderer {
 
     // Add audits without a group first so they will appear first.
     const notAGroup = 'NotAGroup';
+    const hiddenGroup = 'hidden';
```

Comment:
nit: can just inline this string into the `groupId` check, since it's a slightly different thing than `notAGroup`.